src/tree/LocalRootTreeItemBase.ts

```diff
@@ -286,14 +294,13 @@ export abstract class LocalRootTreeItemBase<TItem extends ILocalItem, TProperty
         }
     }
 
-    private getDockerErrorTreeItems(context: IActionContext, error: unknown): AzExtTreeItem[] {
-        const connectionMessage = localize('vscode-docker.tree.failedToConnectMessage', 'Failed to connect. Is Docker installed and running?');
-
-        const result: AzExtTreeItem[] = [
-            new GenericTreeItem(this, { label: connectionMessage, contextValue: 'dockerConnectionError', iconPath: getThemedIconPath('statusWarning') }),
-            new GenericTreeItem(this, { commandId: 'vscode-docker.installDocker', label: localize('vscode-docker.tree.installDocker', 'Install Docker...'), contextValue: 'installDocker', iconPath: getThemedIconPath('docker') }),
-            new OpenUrlTreeItem(this, localize('vscode-docker.tree.additionalTroubleshooting', 'Additional Troubleshooting...'), 'https://aka.ms/AA37qt2'),
-        ];
+    private getDockerErrorTreeItems(context: IActionContext, error: unknown, dockerInstalled: boolean): AzExtTreeItem[] {
+        const result: AzExtTreeItem[] = dockerInstalled
+            ? [
+                new GenericTreeItem(this, { label: localize('vscode-docker.tree.dockerNotRunning', 'Failed to connect. Is Docker running?'), contextValue: 'dockerConnectionError', iconPath: getThemedIconPath('statusWarning') }),
+                new OpenUrlTreeItem(this, localize('vscode-docker.tree.additionalTroubleshooting', 'Additional Troubleshooting...'), 'https://aka.ms/AA37qt2')
+            ]
+            : [new GenericTreeItem(this, { label: localize('vscode-docker.tree.dockerNotInstalled', 'Failed to connect. Is Docker installed?'), contextValue: 'dockerConnectionError', iconPath: getThemedIconPath('statusWarning') })];
```

Comment:
If we want we should be able to make clicking this node run the `installDocker` command. See what @BigMorty thinks.

Reply:
Lets wait for Mike's input in the email thread that I started on this.

Reply:
In stand-up, we agreed on not show the install link here.